src/sinks/datadog/events/mod.rs

```diff
@@ -7,3 +7,5 @@ pub mod sink;
 mod tests;
 
 pub use self::config::DatadogEventsConfig;
+
+const NAME: &str = "datadog_events";
```

Comment:
A name should be macro'd as this is a named component, I don't believe we need this anymore.

Reply:
Oh neat. :thinkies: do you have an example or the name of the macro to call ?

Reply:
Discussed offline, but via `NamedComponent`

Reply:
Awesome, thanks! :tada: